packages/js/email-editor/README.md

```diff
@@ -0,0 +1,85 @@
+**Storage** – we store email content as custom post-type + we use templates (a couple of dummy hardcoded templates) to carry shared parts of the content (header, footer) and style settings. Currently, we store complete style settings per template.
+**Renderer** – responsible for converting saved HTML from Gutenberg editor to HTML for email clients.
+**Theme Controller** – The theme controller is used to generate settings and styles for the editor. We can define which features for working with content are available in settings. The styles are also used in the Render.
+
+### Email Editor
+* Bootstrapped in the plugin in the [email editor controller](https://github.com/mailpoet/mailpoet/blob/13bf305aeb29bbadd0695ee02a3735e62cc4f21f/mailpoet/lib/EmailEditor/Integrations/MailPoet/EmailEditor.php)
+* **Components folder** - basically the whole UI of the editor. Most of Gutenberg’s blocks magic happens in block-editor folder.
+* **Hooks folder** – several custom hooks mostly to help us pass around data and process data from the store or combine them from multiple stores.
+* **Store folder** – classic `wordpress/data` store. We try to use stores from packages we build on as much as possible, but sometimes we need to add an action etc.
+* **Layouts folder** – contains one layout. Gutenberg support flex, grid but these can’t be used because email clients don’t support them. Flex-email is restricted layout that supports some features from flex. It is used for buttons.
+* **Blocks folder** – when we add support for block, we usually need to do some adjustments and hide some styling options. This is done via Blocks API and save in this folder.
+* **Lock-unlock** – key to open the Pandora box with private components from WP packages
+* **.sccs files** – we don’t write much CSS. We load styles for the post editor (this is done in page controller) but sometimes we need some adjustments of have a custom ui
+* **Custom blocks** - MailPoet custom blocks should be built in [mailpoet-custom-email-editor-blocks folder](https://github.com/mailpoet/mailpoet/tree/13bf305aeb29bbadd0695ee02a3735e62cc4f21f/mailpoet/assets/js/src/mailpoet-custom-email-editor-blocks)
+
+### Theme
+Basically theme.json and ThemeController. We don’t use theme.json from the site’s theme. Not every site is using a block theme.
+Our current approach is that for email, we have our own theme and we want to handpick settings from the site’s theme to achieve a similar design as the site. Currently, we pick only the color palette.
+
+### Templates
+There is no official API for plugins to add templates (there will be in WP6.7). For now, we use low level hooks to add email templates, so they can be available for consumption in the email editor (via Rest API) and for the renderer.
+This is done in Templates.php.
+
+### Patterns
+Similar to templates but for patterns. Currently, we have one email pattern with initial email content.
```

Comment:
I'm not sure about this part. It is related to both PHP and JS packages, and also it is more like a description of the current state, which might be outdated soon. I would consider removing it so that we don't have to maintain it.

Reply:
Noted. Fixed in https://github.com/mailpoet/mailpoet/pull/5959/commits/dd8bc0d1aec393585b4e44e7826854fb93ad9b3b